libimage/save.go

```diff
@@ -105,7 +105,7 @@ func (r *Runtime) saveSingleImage(ctx context.Context, name, format, path string
 		destRef, err = ociTransport.NewReference(path, tag)
 		options.ManifestMIMEType = ociv1.MediaTypeImageManifest
 
-	case "docker-dir":
+	case "docker-dir", "dir":
 		destRef, err = dirTransport.NewReference(path)
 		options.ManifestMIMEType = manifest.DockerV2Schema2MediaType
```

Comment:
So actually there’s this hard-coded format setting, making `docker-dir` _not_ be the raw “preserve original format”  destination that c/image `dir:` is.

And the history of that is https://github.com/containers/podman/commit/d2ab53aa5f67cb65a75f2c36fb076c0debd961ca , which has originally used `oci-dir` for `dir:` with OCI conversion forced, and `docker-dir` for `dir:` with v2s2 conversion forced. I.e. the `docker-` part in the name is historically very much intentional and semantically meaningful.

Only fairly recently was `oci-dir` changed to use the OCI repository format ( https://github.com/containers/podman/pull/6814 ), making the `oci-dir`/`docker-dir` less of a clear-cut pair.

Still — right now it is not a “Docker-defined directory”, but it _is_ a “Docker-format-containing directory”. I’m not too happy about dropping the format name from the format.

I’m tempted to complicate this further, leave `docker-dir` as is, and define a new `dir` to have the c/image meaning (i.e. no conversion unless the user _explicitly_ asked for one).

Reply:
> I’m tempted to complicate this further, leave docker-dir as is, and define a new dir to have the c/image meaning (i.e. no conversion unless the user explicitly asked for one).

Actually a `podman push $source dir:…` should work for that already, so we don’t need to make this an explicit `podman image save` feature without any specific user demand.

Reply:
Yup
Man docker save shows none of these. So there is little value in us flooding the man pages with "docker".

Reply:
docker save does not even support --format.

Reply:
Podman does support --format, so Podman users need to know which value to use to interoperate with various tools, whether that is `umoci` or Docker. (Even if the _default_ is to use a Docker-compatible format, and users don’t need to explicitly use `--format` to get that kind of output, I think the man page should document clearly enough that that’s what the command is intended to do, so that users can rely on that.)